cg/cli/demultiplex/copy_novaseqx_data.py

```diff
@@ -0,0 +1,79 @@
+from pathlib import Path
+import os
+import shutil
+from typing import List, Optional
+
+from cg.constants.demultiplexing import DemultiplexingDirsAndFiles
+
+
+def is_copied(analysis_directory: Path) -> bool:
+    return Path(analysis_directory, DemultiplexingDirsAndFiles.COPY_COMPLETE).exists()
+
+
+def is_analyzed(analysis_directory: Path) -> bool:
+    return Path(
+        analysis_directory,
+        DemultiplexingDirsAndFiles.DATA,
+        DemultiplexingDirsAndFiles.ANALYSIS_COMPLETED,
+    ).exists()
+
+
+def is_in_demultiplexed_runs(flow_cell: Path, demultiplexed_runs: Path) -> bool:
```

Comment:
```suggestion
def is_flow_cell_in_demultiplexed_runs(flow_cell: Path, demultiplexed_runs: Path) -> bool:
```

Reply:
Since all this function need is the final component of the path - I think it makes sense to only supply that and be explicit what is expected i.e. is flow_cell.name a dir or a file in this function. 

So calling it in the dir context would be something like:
```
is_flow_cell_in_demultiplexed_runs(flow_cell_dir=flow_cell.name(), demultiplexed_runs_path=demultiplexed_runs) 
```